client/components/Register.jsx

```diff
@@ -0,0 +1,55 @@
+import { connect } from 'react-redux'
+import React from 'react'
+
+class Register extends React.Component {
+  constructor (props) {
+    super(props)
+    this.state = {
+      // bring in photourl into img tag when input photurl typed in
+      photo_url: '/images/Blank_Avatar.png'
```

Comment:
`photo_url` should be `photoUrl`
Rename `Blank_Avatar.png` to `blank-avatar.png`